### PR TITLE
DR7 restructure and PyPI workflow

### PR DESCRIPTION
**Pull request by tapastro**
Opening fresh restructure PR in favor of #72 , due to to updates and PR collisions.

### CODE REVIEW

File: src/ullyses/tests/test_wrapper.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 V_HK_ORI = {'name': 'v-hk-ori',
             'url': 'https://stsci.box.com/shared/static/dzw9gt283sz6rnekhtq88jxgcl2tq376.gz'}
 
-RELEASE = 'dr6'
+RELEASE = __release__
```
Comment (tapastro) [inline src/ullyses/tests/test_wrapper.py:23]: For reviewers: should this match the truth file release? There were previously two "release" values in the tests, one set here as `RELEASE = 'dr6'` and the other import from ullyses_utils as `VERSION`. I may have clobbered the difference by blanket-updating them both with the current package version.

Comment (jotaylor) [reply src/ullyses/tests/test_wrapper.py:23]: Good question. This is probably a question for @stscirij

**Review by jotaylor — APPROVED** [review]
From my perspective, everything works after the couple fixes I just made. (These were on the previous dr7 build branch but after your updates Tyler). The only interesting thing I noticed was that I couldn't get the `ullyses_utils` version.
```
>>> import ullyses_utils
>>> ullyses_utils.__version__
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'ullyses_utils' has no attribute '__version__'
```
However, this probably needs to be fixed in the utils repo, not here.

I will approve, however, Robert should address the test question.

File: src/ullyses/tests/test_wrapper.py (hunk 1 above)
Comment (stscirij) [reply src/ullyses/tests/test_wrapper.py:23]: The dr6 is the name of a subdirectory in the inflated zip file pulled from Box.  When dr7 products are made, the directory structure of the box zip file will be changed to match.  I am going to have to re-make the Box files anyway, so this change is perfectly OK

**Pull request merged**